### ISSUE TITLE
Please implement intents AKA share button

### ISSUE THREAD

**Issue by rainthundermoon** [JonathanCoe/bitseal#8]
https://developer.android.com/guide/components/intents-filters.html

Also known as the share button, I feel this is a very required feature that is missing. I would like to be able to send a chunk of text or a GPG key or such from another program using the intent rather than using copy and paste. I would also like to be able to save messages or say send a message to APG for decryption, which can be done easily using the share button. 

To summarize there should be a share button inside the application and an intent for other applications to interface with bitseal.

**Comment by JonathanCoe** [comment]
I agree that this would be a good thing to implement, so thanks for the suggestion. Unfortunately I have very little spare time available at the moment, so it may be a while before I can do it. If anyone else feels like attempting it they're more than welcome to do so.
